components/products.html

```diff
@@ -0,0 +1,205 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Products | nyaa-kart</title>
+    <link rel="stylesheet" href="../main.css">
+</head>
+
+<body>
+
+    <header class="header">
+        <a href="/"><img src="../images/nyaa-kart.svg" alt="nyaa-UI Logo" class="logo"></a>
+
+
+        <form action="#" class="search">
+            <input type="text" class="search__input" placeholder="search here">
+            <button class="search__button">
+                <svg class="search__icon">
+                    <use href="../images/sprite.svg#icon-magnifying-glass">
+                    </use>
+                </svg>
+            </button>
+        </form>
+
+        <nav class="user-nav">
+            <div class="user-nav__icon-box">
+                <svg class="user-nav__icon">
+                    <use href="../images/sprite.svg#icon-heart-outlined">
+                    </use>
+                </svg>
+
+                <span class="user-nav__notification">5</span>
+            </div>
+
+            <div class="user-nav__icon-box">
+
+                <svg class="user-nav__icon">
+                    <use href="../images/sprite.svg#icon-shopping-cart">
+                    </use>
+                </svg>
+
+                <span class="user-nav__notification">18</span>
+            </div>
+
+            <div class="user-nav__user">
+                <span class="user-nav__user-name user-nav__login">Login</span>
+            </div>
+        </nav>
+    </header>
+
+    <div class="product-container">
+
+        <div class="product-card">
+            <div class="product-card__cover">
+                <img class="img" src="../images/product.png" alt="">
+                <svg class="product-card__icon">
+                    <use href="../images/sprite.svg#icon-heart">
+                    </use>
+                </svg>
+            </div>
+
+            <div class="product-card__content">
+                <div class="product-card__title">Men Premium Jacket</div>
+                <h3 class="product-card__price">$2000</h3>
+
+                <button class="btn">Add to Card</button>
+
+            </div>
+        </div>
+
+        <div class="product-card">
+            <div class="product-card__cover">
+                <img class="img" src="../images/product.png" alt="">
+                <svg class="product-card__icon">
+                    <use href="../images/sprite.svg#icon-heart">
+                    </use>
+                </svg>
+            </div>
+
+            <div class="product-card__content">
+                <div class="product-card__title">Men Premium Jacket</div>
+                <h3 class="product-card__price">$2000</h3>
+
+                <button class="btn">Add to Card</button>
+
+            </div>
+        </div>
+        <div class="product-card">
+            <div class="product-card__cover">
+                <img class="img" src="../images/product.png" alt="">
```

Comment:
let there be some descriptive text for image in alt attribute.

Reply:
thanks, fixed